pghoard/rohmu/object_storage/google.py

```diff
@@ -25,7 +25,7 @@
 import httplib2
 from googleapiclient.discovery import build
 from googleapiclient.errors import HttpError
-from googleapiclient.http import (MediaFileUpload, MediaIoBaseDownload, MediaIoBaseUpload, MediaUpload)
+from googleapiclient.http import (build_http, MediaFileUpload, MediaIoBaseDownload, MediaIoBaseUpload, MediaUpload)
```

Comment:
This looks like the natural place for this import but apparently static checker would like it to be at the end